templates/rhpam73-authoring-ha.yaml

```diff
@@ -1189,8 +1189,10 @@ objects:
             value: "${APPLICATION_NAME}-mysql"
           - name: RHPAM_SERVICE_PORT
             value: "3306"
+          - name: TIMER_SERVICE_DATA_STORE
+            value: "${APPLICATION_NAME}-mysql"
           - name: KIE_SERVER_PERSISTENCE_DIALECT
-            value: "org.hibernate.dialect.MySQLDialect"
+            value: "org.hibernate.dialect.MySQL5Dialect"
```

Comment:
@sutaakar  Ha, I found this error myself when creating the datagrid ha authoring templates. ::) Should we be even more specific, and replace with MySQL**57**Dialect? Or do you feel MySQL**5**Dialect is sufficient?

Reply:
Well, that is the question, on premise we currently test with MySQL5InnoDBDialect, however I think we can use MySQL57Dialect as it is the most specific one.

Reply:
I wonder if we should parameterize the dialect, but default it to MySQL57Dialect to best align with the default that's already there for MYSQL_IMAGE_STREAM_TAG (which is 5.7). But this way, people can change to MySQL5InnoDBDialect if they want. Thoughts?

Reply:
Sounds good to me.
I guess this should be reflected on all templates using database, right?

Reply:
I do agree we should use 5.7 as it is the default as it match the mysql version we use on OpenShift.
The dialect already is configurable, do you mean set something like MySQL${VERSION}InnoDBDialect?

Reply:
Using version placeholder isn't ideal as there is no specific dialect for every database version. For MySQL there is MySQL5Dialect, MySQL55Dialect and MySQL57Dialect.